### PR TITLE
Add delay option to data_generator operation

### PR DESCRIPTION
**Pull request by sotojn**
This PR makes the following changes:

### New Features
- Adds new `delay` option to the `data_generator` operation 
  - This option allows the user to specify how long it will take for the worker to complete its slice.
  - _Example:_ If **delay** is set to `60` and **size** is set to `10000` with only 1 worker, the job will generate 10,000 records per minute approximately. You can achieve about the same record output with more slices by decreasing size and decreasing delay.
### Documentation 
- Bumps **standard-assets** from `v1.0.3` to `1.1.0`
- Updated docs to include use case and description for `delay` param in `data_generator`

#898

### CODE REVIEW

File: asset/src/data_generator/schema.ts [hunk 1]
```diff
@@ -74,6 +78,18 @@ export default class Schema extends ConvictSchema<DataGenerator> {
                 default: false,
                 format: Boolean
             },
+            delay: {
+                doc: 'Time in seconds that a worker will complete a slice. Great'
```
Comment (busma13) [inline asset/src/data_generator/schema.ts:82]: Maybe say time the worker waits after generating records before returning slice?

**Review by busma13 — APPROVED** [review]
Looks good. I would reword the description, but it's not necessary if you like how you explained it.

**Review by godber — APPROVED** [review]
